### PR TITLE
test

### PR DESCRIPTION
**Pull request by ehalsmer**

### CODE REVIEW

**Comment by ehalsmer** [comment]
This is not yet ready for review

**Comment by ehalsmer** [comment]
Last commit has the completed CSS

File: great-idea-website/css/index.css [hunk 1]
```diff
@@ -65,6 +65,100 @@ h1, h2, h3, h4, h5 {
...
 .navigation {
-	color:red;
+	margin:  30px 0px;
+	display: inline-block;
+	width: 693px;
+	box-sizing: border-box;
+}
+
+.navigation a {
+	text-decoration: none;
```
Comment (AnnaTownsend9916) [inline great-idea-website/css/index.css:81]: Nice job getting rid of text decoration!

File: great-idea-website/index.html [hunk 2]
```diff
@@ -16,53 +16,67 @@
...
   </section>
  
   <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
   
   <section class="bottom-content">
-      Services
-      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      <div>
+        <h3>Services</h3>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
```
Comment (AnnaTownsend9916) [inline great-idea-website/index.html:59]: Make sure to put the text in a paragraph

File: great-idea-website/index.html [hunk 3]
```diff
@@ -16,53 +16,67 @@
...
+
+    <hr />
+
 
   <section class="contact">
-      Contact
-      123 Way 456 Street
-      Somewhere, USA
-      1 (888) 888-8888
+      <h3>Contact</h3>
```
Comment (AnnaTownsend9916) [inline great-idea-website/index.html:76]: Try to make this a footer :)

File: great-idea-website/index.html [hunk 4]
```diff
@@ -16,53 +16,67 @@
...
+      123 Way 456 Street<br />
+      Somewhere, USA<br /><br />
+      1 (888) 888-8888<br /><br />
       sales@greatidea.io
   </section>
```
Comment (AnnaTownsend9916) [inline great-idea-website/index.html:82]: Overall very nice work for your first day, just a couple of small things to add. 👍

**Pull request merged**